index.html

```diff
@@ -0,0 +1,114 @@
+            </span> PFolio
+        </h1>
+        <ul id="main-nav" class="nav_bar">
+            <li><a href="#home">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact Us</a></li>
+        </ul>
+    </nav>
+    <!-- Header -->
+    <header id="heading">
+        <div class="header-wrap">
+            <h2 class="l-heading">Welcome to My Website</h2>
+            <p class="lead">Just my Portfolio</p>
+            <a href="#about" class="btn">Read More</a>
+        </div>
+    </header>
+
+    <!--section -->
+    <section id="about" class="bg-light py-1">
+        <!-- Work Experience-->
+        <div class="job">
+            <div class="job-heading">
+                <h2 class="m-heading text-center">
+                    <span class="text-primary">Work</span> Experience
+                </h2>
+            </div>
+            <div class="item">
+                <div class="job-item">
+                    <h3>Dufuna CodeCamp</h3>
+                    <p>Programming Scholar<br>
+                        October 2018 - March 2020</p>
+                </div>
+                <div class="job-item">
+                    <h3>Konga</h3>
+                    <p>FrontEnd developer<br>
+                        March 2020 - Till Present</p>
+                </div>
+            </div>
+            <section id="content">
+                <div class="content-img"></div>
+                <div class="content-text bg-dark p-1">
+                    <h1 class="m-heading text-center"><span class="text-primary">About</span> Me
+                    </h1>
+                    <P class="sub">Professional Path</P>
+                    <p class="bio">
+                        I am Idayat Abdulkareem,software developer. A graduate of Unilorin<br>
+                        currently working with Konga as a FrontEnd Developer.<br>
+                        I started coding six months ago and I enjoy coding.
+                    </p>
+                </div>
+        </div>
+        </div>
+        </div>
+    </section>
+    <!-- section contact-->
+    <section id="contact">
+        <div class="contact-list">
+            <div class="contact-email">
+                <i class="fas fa-envelope"></i> Email<br>
+                <div class="contact-details">idayat.abdulkareem@konga.com</div>
+            </div>
+            <div class="contact__phone">
+                <i class="fas fa-mobile-alt"></i> Phone<br>
+                <div class="contact-details">(+234) 081-094-33993</div>
+            </div>
+            <div class="contact__address">
+                <i class="fas fa-location-arrow"></i> Address<br>
+                <div class="contact-details">33 Airport Road, Off Ajao-Estate. Lagos.</div>
+            </div>
+        </div>
+    </section>
+    <section id="icon" class="py-1">
+        <div class="social-icon" class="py-1">
+            <a href="https://twitter.com" target="blank">
```

Comment:
Note that when you use target="_blank", always add rel="noopener" or rel="noreferrer". Visit https://developers.google.com/web/tools/lighthouse/audits/noopener for more information.